packages/ui/src/services/component-config/utils/use-component-config-props.ts

```diff
@@ -1,31 +1,62 @@
-import type { VuesticComponent, VuesticComponentName, Props } from '../types'
+import { VuesticComponentName, Props, VuesticComponent } from '../types'
 import { useLocalConfig } from '../../../composables/useLocalConfig'
 import { useGlobalConfig } from '../../global-config/global-config'
 import { computed } from 'vue'
-import { injectChildPresetPropFromParent } from '../../../composables/useChildComponents'
+import { injectChildPropsFromParent } from '../../../composables/useChildComponents'
+import { ComponentPresetProp, PresetPropValue } from '../../../composables'
+import { notNil } from '../../../utils/isNilValue'
+import { head } from 'lodash'
```

Comment:
```suggestion
import head from 'lodash/head'
```

Otherwise, we import whole lodash package unfortunately...

Reply:
Good thing would be alias lodash with lodash-es which is tree-shakable

Reply:
Ideally, we want to remove lodash at all.